webrtc.html

```diff
@@ -2530,23 +2539,55 @@
 
           <dd>
             <p>
-              Indicates that this encoding is actively being sent.
-              Setting it to false causes this encoding to no longer be sent.
-              Setting it to true causes this encoding to be sent.
+              If true, this encoding will be sent if sufficient bandwidth if available.
+              If false, this encoding will not be sent.
             </p>
           </dd>
 
           <dt>double priority</dt>
           <dd>
             <p>
-              Indicates the relative priority of this encoding, across
-              all RtpSenders of a given PeerConnection.  When there is
+              The relative priority of this encoding across all
+              RtpSenders of a given PeerConnection.  When there is
               limited bandwidth available to a PeerConnection, higher
               prioirty encodings will be sent with more bandwidth, and
               lower priority encodings will be sent with less
               bandwidth.
```

Comment:
This is being updated in https://github.com/w3c/webrtc-pc/pull/259.  You may like that version better.

And that really has nothing to do with this PR.  It looks like I accidentally changed some grammar/formatting.